templates/Entry/entries.html.twig

```diff
@@ -1,5 +1,8 @@
 {% extends "layout.html.twig" %}
 
+{% set has_filters = form is not null %}
```

Comment:
check on `is_granted('LIST_ENTRIES')` got removed because now all actions using this template are protected by the security check

Reply:
kept the `form is not null` condition because it's false on the `showEntriesForTagAction`